test/model/unreads_test.dart

```diff
@@ -190,13 +190,13 @@ void main() {
         eg.streamMessage(stream: stream, topic: 'c', flags: []),
       ]);
       check(model.countInStream      (stream.streamId)).equals(5);
```

Comment:
nit: preserve alignment (here and below)
```suggestion
      check(model.countInStream       (stream.streamId)).equals(5);
```